### PR TITLE
Add CUDA 10.0 build

### PR DESCRIPTION
**Pull request by jakirkham**
Builds the `nvcc` (shim compiler) package for CUDA 10.0.

Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jakirkham** [comment]
Oops! That should have been a draft. Marked as WIP.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "nvcc" %}
-{% set version = environ.get("CUDA_VER", "9.2") %}
+{% set version = environ.get("CUDA_VER", "10.0") %}
```
Comment (kkraus14) [inline recipe/meta.yaml]: Should we default to 9.2 instead? Things are generally forward compatible but not necessarily backwards compatible.

Comment (jakirkham) [reply recipe/meta.yaml]: The default here is only to make the linter happy. This is always set to `10.0` by the build.

ref: https://github.com/conda-forge/staged-recipes/pull/8229#pullrequestreview-275703107

**Comment by jakirkham** [comment]
> Can you make the two versions build from the recipe? You can use a list for `docker_image`

Yeah I'd prefer that actually, but had difficulty getting this to work when I tried it previously. Do you have a suggestion on how to implement? If so, would you be willing to throw up a quick PR? 🙂

**Comment by isuruf** [comment]
You need to replace `version` with `CUDA_VER` and then rerender

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
Sorry maybe I'm misunderstanding your suggestion, @isuruf, what still needs to be done here? If you feel comfortable making the change, would welcome it in this PR 🙂

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Comment by isuruf** [comment]
@jakirkham, can you manually render?

**Comment by jakirkham** [comment]
Superseded by PR ( https://github.com/conda-forge/nvcc-feedstock/pull/7 ).

**Pull request closed without merging**